pkg/config/piped_test.go

```diff
@@ -230,6 +230,21 @@ func TestPipedConfig(t *testing.T) {
 						},
 						{
 							Name: "integration-slack-api-with-mentioned-accounts",
+							Slack: &NotificationReceiverSlack{
+								OAuthToken:        "token",
+								ChannelID:         "testid",
+								MentionedAccounts: []string{"user1", "user2"},
+							},
+						},
+						{
+							Name: "integration-slack-api-with-oauthTokenData",
+							Slack: &NotificationReceiverSlack{
+								OAuthTokenData: "token",
+								ChannelID:      "testid",
+							},
+						},
+						{
+							Name: "integration-slack-api-with-oauthTokenData-and-mentioned-accounts",
```

Comment:
Doesn't need OAuthTokenFile case ?

Reply:
Hi @sivchari  Thank you so much for your checking! I added 2 case with `OAuthTokenFile`